elastic/kibana
# [ResponseOps][Alerting] Improve ES Query reason message

Resolves https://github.com/elastic/kibana/issues/166984

## Summary

Updating the reason message to align more with the other rule types. Updating the format to be more like this:
`Document count is 18 in the last 1 min for host-0 in APM data view. Alert when above 10.
`

### Checklist



### To verify

- Create an es query rule and set the scope to Metrics, so it's visible in Observability
- Verify that the reason message is correct for the 3 different types of es query rule
